manual/C/ch_Finance-Quote.xml

```diff
@@ -240,6 +251,199 @@ SYNOPSIS
     </procedure>
   </sect1>
 
+  <sect1 id="finance-quote-helper">
+    <title>Helper Scripts for Update and Diagnostics</title>
+
+    <abstract>
+      <para>&app; provides several utilities that support you in your daily work with &app-fq;.
+        So there is a script with which you can check your symbols for the securities. Other
+        scripts perform the initial installation or keep &app-fq; up to date.
+        &app-fq; software module. These programs have names that begin with gnc-fq-*, and are
+        themselves written in the &app-perl; programming language. 
+      </para>
+    </abstract>
+
+    <procedure>
+      <title>Running &app-perl; Scripts Under &win;</title>
+
+      <step>
+        <simpara>Open a <application>CMD</application> or <application>Powershell</application>
+          window: Click <guimenu>Start</guimenu> and type either
+          <userinput>cmd</userinput> or <userinput>powershell</userinput>
+          then select the resulting menu item. You don't need to run as administrator.
+        </simpara>
+      </step>
+
+      <step>
+        <simpara>Change to the drive containing the &app; program files —usually C drive—
+          if that is not already the current drive.
+        </simpara>
+      </step>
+
+      <step>
+        <simpara>Since, as previously recommended, the GnuCash directory was entered in the <envar>PATH</envar>
+          envinronment variable, prefix any gnc-fq-xxx &app-perl; script command with
+          <quote>Perl</quote> and a space, e.g. <userinput>perl gnc-fq-check</userinput>.
```

Comment:
This is wrong. Perl needs to be on the path, otherwise you have to specify the full path to perl.exe. The default installation is IIRC (not at a windows box ATM) `C:\strawberry-perl\bin\perl.exe`. Unless you've changed directory to GnuCash's `bin` directory or added it to `%PERLLIB` (expert stuff, don't write anything about that!) you must specify the full path to run these on Windows:
`<userinput><replaceable>C:\strawberry-perl\bin\</replaceable>perl.exe <replaceable>C:\Progarm Files (x86)\gnucash\bin\</replaceable>gnc-fq-check</userinput>`